README.md

```diff
@@ -64,20 +64,20 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: suzuki-shunsuke/github-action-renovate-config-validator@v0.1.2
+      - uses: suzuki-shunsuke/github-action-renovate-config-validator@v1
```

Comment:
```suggestion
      - uses: suzuki-shunsuke/github-action-renovate-config-validator@v1.0.1
```

There is no tag `v1`.

Reply:
There should be a v1 following the minor/patch releases!

Docs: https://docs.github.com/en/actions/creating-actions/about-custom-actions#using-tags-for-release-management (point 3)